testnets/holesky/README.md

```diff
@@ -407,16 +407,9 @@ After all of the steps above have been completed, we can proceed with running th
 
 There are various way to run the Bolt Sidecar depending on your preferences and your preferred signing methods:
 
-- Docker mode (recommended)
-- [Commit-Boost](https://commit-boost.github.io/commit-boost-client) mode
-  (requires Docker)
-- Native mode (advanced, requires building everything from source)
-
-Running the Bolt sidecar as a standalone binary requires building it from
-source. Both the standalone binary and the Docker container requires reading
-signing keys from [ERC-2335](https://eips.ethereum.org/EIPS/eip-2335) keystores,
-while the Commit-Boost module relies on an internal signer and a custom PBS
-module instead of regular [MEV-Boost](https://boost.flashbots.net/).
```

Comment:
I felt like this part was confusing so I removed it. It's explained more in detail below anyway.